arena.js

```diff
@@ -0,0 +1,145 @@
+/* arena.js */
+
+class Arena {
+	constructor(name) {
+		this.name = capitalize(name);
+		this.gladiators = [];
+	}
+	addGladiator(gladiator) {
+		console.log(`Welcome to the arena, ${gladiator.name}.`);
+		this.gladiators.push(gladiator);
```

Comment:
I definitely should add a check there, but the reason it worked for me was that line 12 runs the fight sequence and, as a result, brings the length of the array back down to 0 or 1